docs/getting-started/README.md

```diff
@@ -0,0 +1,95 @@
+    - Clone this repo locally -  we will come back to this repo later.
+## Configure the cluster
+
+Now that we have our cluster ready, we need to setup our getting-started namespace and RBAC. We will keep everything inside this single namespace for easy cleanup. In the unlikely event that you get stuck/flummoxed, the best course of action might be to just delete this namespace and start fresh.
+
+- Create the *getting-started* namespace, where all our resources will live:
+  - `kubectl create namespace getting-started`
+- [Create the admin user, role and rolebinding](./rbac/admin-role.yaml)
+  - `kubectl apply -f ./docs/getting-started/rbac/admin-role.yaml`
+- [Create the create-webhook user, role and rolebinding](./rbac/webhook-role.yaml)
+  - `kubectl apply -f ./docs/getting-started/rbac/webhook-role.yaml`
+  - This will allow our webhook to create the things it needs to.
+
+## Install the Pipeline and Triggers
+
+Now we have to install the Pipeline we plan to use and also our Triggers resources.
+
+Our Pipeline will do a few things:
+- Retrieve the source code
+- Build and push the source code into a docker image
+- Push the image to the specified repository
+- Run the image locally
+<!-- -->
+- [Install the Pipeline](./pipeline.yaml)
+  - The Pipeline will build a docker image with img and deploy it locally via kubectl image.
+  - `kubectl apply -f ./docs/getting-started/pipeline.yaml`
+<!-- -->
+Our Triggers project will pickup from there:
+- We will setup an EventListener to accept and process Github Push events
+- A TriggerTemplate to create templates Resources and PipelineRun per event received.
+<!-- -->
+- [Install the TriggerTemplate, TriggerBinding and EventListener](./triggers.yaml)
+  -  First, **edit** the `triggers.yaml` file to reflect
+    - The Docker repository to push the image blob
+      - You will the `DOCKERREPO-REPLACEME` string everywhere it is needed.
+  - Once you have updated the triggers file, you can apply it!
+  - `kubectl apply -f ./docs/getting-started/triggers.yaml`
+  - If that succeeded, your cluster is ready to start handling Events.
+
+## Configure Github Webhook
+
+Now we need some events to work with. We need to configure Github to send events to our Cluster, which is quick but has a few tricky parts.
+
+- Establish your inbound EXTERNAL-URL hostname:
+  - In order to accept Github webhook payloads, our listener must be reachable from the internet.
+  - Our EventListener will create a [Loadbalancer Service](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) for us, so we just need to determine its hostname. How the loadbalancer gets created will depend on how you are deploying this getting-started - AWS will behave differently than GCP, and so on.
+  - Inspect the output of `k get service/getting-started-listener -n getting-started -o yaml` - there should be a hostname associated with the ingress now.
```

Comment:
Not everyone has `k` as an alias to `kubectl`. Also, `kubectl` is used above and simplifies copy-pasting.